pom.xml

```diff
@@ -193,11 +193,16 @@
                 <version>3.11.0</version>
 
                 <configuration>
-                    <release>${java.version}</release>
```

Comment:
Toto neviem ake bude mat dopady, ale s compiler flags to neslo. cc @sabomichal ?

Reply:
Trochu mi chyba kontext, ale [dokumentacia](https://maven.apache.org/plugins/maven-compiler-plugin/examples/set-compiler-release.html) hovori, aky je rozdiel medzi source/target a release flagom. Konkretne tuto to zrejme narazalo na tuto vec (aj ked vy ste zrejme vzdy na build pouzivali jdk vo verzii 17):

_"Unlike the [-source and -target options](https://maven.apache.org/plugins/maven-compiler-plugin/examples/set-compiler-source-and-target.html), the compiler will detect and generate an error when using APIs that don't exist in previous releases of Java SE."_

Kazdopadne, ako pozeram tak property java.version chcete mat asi pouzite rovnako aj pri source/target flagoch miesto hardcoded hodnoty ako je tam teraz.

Reply:
@celuchmarek toto asi fixnime tiez po ceste